Makefile

```
@@ -322,11 +321,16 @@ telemetrygen:
 
 .PHONY: update-otel
 update-otel:$(MULTIMOD)
-	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m stable --commit-hash $(OTEL_STABLE_VERSION)
-	git add . && git commit -s -m "[chore] multimod update stable modules"
-	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m beta --commit-hash $(OTEL_VERSION)
-	git add . && git commit -s -m "[chore] multimod update beta modules"
+	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector-contrib -m stable 
```

```suggestion
	$(MULTIMOD) sync -s=true -o $(SRC_ROOT) -m stable 
```
I'm wondering if using this would allow us to be flexible with local directory naming, to enable worktrees.

(Same comment for other usage)

---

actually this works very well - i tested it locally and it seems to be a more sustainable solution then filling the directory itself
-> changed in recent commit